hack/parallel.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+NAMESPACE=${KUBEVIRT_INSTALLED_NAMESPACE:-kubevirt}
+
+kubectl() { cluster-up/kubectl.sh "$@"; }
+
+kubectl get nodes >/dev/null 2>&1
+if [ $? -ne 0 ]; then
+    echo "Cluster not found, exiting"
+    exit 1
+fi
+
+kubectl get ns $NAMESPACE >/dev/null 2>&1
+STATUS=$?
+
+if [ $STATUS -eq 0 ]; then
+    echo "Kubevirt namespace found, cleaning"
+    ./hack/cluster-clean.sh >/dev/null 2>&1 &
```

Comment:
Writing the output to `/dev/null` can hide errors in this process. Please write it to some log or buffer that will be written to STDOUT when ready.